ssh/main.go

```diff
@@ -2,90 +2,74 @@ package main
 
 import (
 	"context"
-	"crypto/rand"
-	"crypto/rsa"
 	"encoding/json"
 	"fmt"
 	"net/http"
 	"time"
 
 	"github.com/gorilla/mux"
+	"github.com/shellhub-io/shellhub/ssh/pkg/tunnel"
+	"github.com/shellhub-io/shellhub/ssh/server"
+	"github.com/shellhub-io/shellhub/ssh/session"
+	log "github.com/sirupsen/logrus"
 	"golang.org/x/net/websocket"
-
-	client "github.com/shellhub-io/shellhub/pkg/api/internalclient"
-	"github.com/shellhub-io/shellhub/pkg/httptunnel"
-	"github.com/sirupsen/logrus"
 )
 
-var magicKey *rsa.PrivateKey
-
-type Options struct {
-	Addr           string
-	Broker         string
-	ConnectTimeout time.Duration
-}
+const (
+	ServerRouterAddress = ":8080"
+	ServerSSHAddress    = "tcp://emq:1883"
+	ServerSSHBroker     = ":2222"
+)
 
 func main() {
-	tunnel := httptunnel.NewTunnel("/ssh/connection", "/ssh/revdial")
-	tunnel.ConnectionHandler = func(r *http.Request) (string, error) {
-		return r.Header.Get(client.DeviceUIDHeader), nil
-	}
-	tunnel.CloseHandler = func(id string) {
-		if err := client.NewClient().DevicesOffline(id); err != nil {
-			logrus.Error(err)
+	tunn := tunnel.NewTunnel("/ssh/connection", "/ssh/revdial")
```

Comment:
`tunn` means nothing to me ... please use `tunnel` so it is easier to understand.